app/controllers/api/v1/user_availabilities_controller.rb

```diff
@@ -16,10 +16,19 @@ def destroy
     render json: timeblock
   end
 
+  def multiple
+    ids = params[:time_block_ids]
+    p ids
+    ids.each do |id|
+      new_avail = UserAvailability.new(user: current_user, time_block_id: id)
+      new_avail.save
```

Comment:
why not directly `UserAvailability.create()` ? 
you could then fit in one line

Reply:
bad habit I guess